$OpenBSD: patch-libaccounts-glib_ag-account_h,v 1.1.1.1 2013/03/29 13:16:17 jasper Exp $

Don't redefine AgAccount which is already declared in ag-types.h.

--- libaccounts-glib/ag-account.h.orig	Fri Mar 29 13:54:02 2013
+++ libaccounts-glib/ag-account.h	Fri Mar 29 13:54:06 2013
@@ -40,7 +40,6 @@ G_BEGIN_DECLS
 
 typedef struct _AgAccountClass AgAccountClass;
 typedef struct _AgAccountPrivate AgAccountPrivate;
-typedef struct _AgAccount AgAccount;
 
 /**
  * AgAccountClass:
